frontend/src/scenes/instance/SystemStatus/BackupTab.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import { Button, Card, Input, Select, Tooltip } from 'antd'
+import { useActions, useValues } from 'kea'
+import { systemStatusLogic } from 'scenes/instance/SystemStatus/systemStatusLogic'
+import Paragraph from 'antd/lib/typography/Paragraph'
+
+export function BackupTab(): JSX.Element {
+    const { systemStatus } = useValues(systemStatusLogic)
+    const { createBackup, restoreFromBackup } = useActions(systemStatusLogic)
+    const [ restoreCandidate, setRestoreCandidate] = useState('')
+    const [ backupSuffix, setBackupSuffix] = useState('')
+    const defaultSuffix = "ui"
+
+    return (
+        <>
+            <Card>
+                {console.log("In backup tab")}
+                {systemStatus?.backup.is_enabled ? (
+                    <>
+                        <Tooltip title={`Only alphanumeric-_ are accepted. Default suffix is \"${defaultSuffix}\"`}
+                        >
+                            <Input 
+                                style={{ width: 400 }}
+                                addonBefore="YYYY-MM-DD-HH-mm-" 
+                                placeholder="Optional custom suffix" 
+                                onChange={(value) => setBackupSuffix(value.target.value.replace(/[^\w\-]/g, ''))}
+                            />
+                        </Tooltip>
+                        &nbsp;&nbsp;
+                        <Button
+                            type="primary"
+                            onClick={() => createBackup(`${new Date().toISOString().replace(/[:T]/g, '-').substr(0,17)}${backupSuffix?backupSuffix:defaultSuffix}`)}
+                        >
+                            Create Backup
+                        </Button>
+                        <br/> 
+                        <h2 style={{ color: 'var(--danger)' }}>
+                            Danger Zone
+                        </h2>
+                        <Paragraph type="danger">
+                            This is owerwrites all data in Clickhouse. Please be certain.
+                            <br/>
+                        </Paragraph>
+                        <Select 
+                            showSearch
+                            style={{ width: 400 }}
+                            placeholder="Search to Select"
+                            filterOption={(input, option) =>
+                                  option.children.toLowerCase().indexOf(input.toLowerCase()) >= 0
+                            }
+                            onChange={(value) => setRestoreCandidate(value)}
```

Comment:
You want to add an alert here as well for this flow with e.g. a `write confirm to confirm` prompt.

Reply:
Change of plans on how things work, this will be a safe operation after all as well restore only if the table didn't exist.